src/server/seeds/development/03_channels.js

```diff
@@ -0,0 +1,80 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('channels')
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex('channels').insert([
+        {
+          id: 1,
+          title: 'Charles',
```

Comment:
Can you give more relevant names, fx kitchen, front desk, also give some channels a title of null